app/scripts/views/open-view.js

```diff
@@ -107,7 +111,12 @@ const OpenView = Backbone.View.extend({
 
     focusInput: function() {
         if (!FeatureDetector.isMobile) {
-            this.inputEl.focus();
+            if (
+                (FeatureDetector.isDesktop && Launcher.isAppFocused()) ||
```

Comment:
Let's put web/desktop logic to the component and return a value from isFocused. Looks like conceptually there's no difference between web and desktop, we just get the property in a different way.

Reply:
What component do you mean? The launcher (there currently is no web launcher, right?) or the open view?

Reply:
"browser-focus", that was added here :)

Reply:
Ah, allright. Done that. 😀